### DISCUSSION TITLE
Disable Interactivity?

### DISCUSSION THREAD

**Discussion by TaPumpa** [apexcharts/apexcharts.js#3111] [Q&A]
Hello, I came here for help.
I can't find any informations, how to disable interactivity in vanila javascript.

Current code:

for (let i = 0; i < 3; i++) {
                        let stock = eventData.stocks[i];
                        let selchart = "chart_" + stock.name;

                        var options = {
                            chart: {
                                height: 200,
                                type: "area"
                            },
                            dataLabels: {
                                enabled: false
                            },
                            series: [
                                {
                                name: "History of price",
                                data: [stock.price40m, stock.price30m, stock.price20m, stock.price10m, stock.c_price]
                                }
                            ],
                            fill: {
                                type: "gradient",
                                gradient: {
                                shadeIntensity: 1,
                                opacityFrom: 0.7,
                                opacityTo: 0.9,
                                stops: [0, 90, 100]
                                }
                            },
                            xaxis: {
                                categories: [
                                    "40m ago",
                                    "30m ago",
                                    "20m ago",
                                    "10m ago",
                                    "Current price"
                                ]
                            }
                        };

                        let chart = new ApexCharts(document.querySelector(("." + selchart)), options);
                        chart.render();
                    }

Can anyone help me out?
Best Regards

**Comment by robclancy** [comment]
Here is an example where I was able to disable it because I'm just using this as an easy way to make a circle rating SVG.

```js
this.drChart = {
  chart: {
    type: 'donut',
  },
  series: [
    this.data.dr['domain_rating'],
    100 - this.data.dr['domain_rating'],
  ],
  colors: [
    '#7362BF',
    '#eaecee',
  ],
   plotOptions: {
    pie: {
      donut: {
        size: '50%',
      },
      expandOnClick: false,
    }
  },
  legend: { show: false },
  dataLabels: { enabled: false },
  tooltip: { enabled: false },
  states: {
    hover: { filter: { type: 'none ' } },
    active: { filter: { type: 'none ' } },
  },
};
```

**Comment by janstieler** [comment]
Hi,
what interactivity excatly. you can disable some of the inetractivities with setting a bool like on zoom
```
zoom: {
    enabled: false
}
```
a running example on codepen or codesandbox is also helpfull.

Cheers
